R/df-helpers.R

```diff
@@ -59,10 +65,16 @@ gather_template_id_individual <- function(species) {
 }
 
 ## gather_template_ids helper
-gather_template_id_biospecimen <- function(species) {
+gather_template_id_biospecimen <- function(species, biospecimen_type) {
   templates <- get_golem_config("templates")$biospecimen_templates
```

Comment:
Having never worked with golem not sure this is relevant, but should templates be an input parameter and then `get_golem_config` called from the function `gather_template_id_biospecimen`? I'm thinking about how I integrate `config` with function arguments.

Reply:
I'm not exactly sure what you mean. `config` and `golem` (`get_golem_config()` calls `config::get()`) tend to assume a single list of options, not nested lists. However, it is a good point that I could grab the list of templates from the config in `gather_template_ids()` and pass it along, instead of grabbing it multiple times. I'll update this to be a little more efficient.

Reply:
Updated and made it a little nicer (IMO) for how I'm referencing specific lists of templates.

Reply:
Your updated code is spot on with my comment. Looks good!